ppr-ui/src/composables/userAccess/useUserAccess.ts

```diff
@@ -324,6 +328,90 @@ export const useUserAccess = () => {
     }
   }
 
+  /**
+   * Submits a request to create or update a Qualified Supplier record.
+   * @returns {Promise<MhrQsPayloadIF>} - A promise that resolves to the created or updated payload.
+   */
+  const submitQsLawyerNotaryOrDealer = async (): Promise<MhrQsPayloadIF> => {
+    // Build payload for submission
+    const payload: MhrQsPayloadIF = {
+      ...cleanEmpty(getMhrQsInformation.value) as MhrQsPayloadIF,
+      authorizationName: getMhrQsAuthorization.value.authorizationName,
+      phoneNumber: fromDisplayPhone(getMhrQsInformation.value.phoneNumber)
+    }
+
+    // Check for current QS Record
+    const hasQsRecord = await getQualifiedSupplier()
+
+    // Create or Update based on previous record
+    return hasQsRecord
```

Comment:
So we are making api cal `getQualifiedSupplier` to check for QS record and then we making another api call to either `update` to `create` qualified supplier. UI is controlling this logic? Should this be done on backend, or we are reusing existing api's? cc: @doug-lovett

Reply:
Are these questions for me or for Doug?

Reply:
I can answer what i know. 

We are making a call to see if the record exists for the user, yes (either qs or mf). If it exists we will update it, if not we will create it. 
Yes the UI is doing this, as you can see. 

Should we be doing this in only one request? I'm not sure, i don't think so. 
Doug could probably answer much more clearly what best practices are but in my opinion isolated resources is probably a better way to follow the REST principles.

Reply:
Yea, I can see the UI is doing multiple get/post calls, which ideally should be only one post submission call, and updated or create should be done on the backend.

Reply:
A POST should create, a PUT should update. 
In my opinion, i don't think a POST should do both.
 
Is that what your asking?

Reply:
Yes, I agree, I just think it should be handled on the backend (as it's more of a business logic) rather than driven by the UI. But I can get used to this approach as well.